backend/services/deployments/api/http/api_deployments.go

```diff
@@ -310,6 +310,53 @@ func getReleaseOrImageFilter(r *http.Request, version listReleasesVersion,
 	return filter
 }
 
+func getImageFilter(c *gin.Context, paginated bool) *model.ImageFilter {
+
+	q := c.Request.URL.Query()
+
+	names := c.QueryArray(ParamName)
+
+	var exactNames []string
+	var nameprefixes []string
+
+	for _, name := range names {
+		if strings.HasSuffix(name, "*") {
```

Comment:
Maybe add the possibility to escape the last character?
```suggestion
		if strings.HasSuffix(name, `\*`) {
			name = strings.TrimSuffix(name, `\*`) + "*"
			exactNames = append(exactNames, name)
		} else if strings.HasSuffix(name, "*") {
```